src/components/Modal/Modal.js

```diff
@@ -174,6 +174,7 @@ class Modal extends React.Component {
           id={id}
           className={className}
           {...rest}
+          title=""
```

Comment:
This looks wrong. Maybe we can just remove the `title` prop from `Window`.

Reply:
Will try